.github/actions/bouf/action.yaml

```diff
@@ -7,32 +7,37 @@ inputs:
   gcpServiceAccountName:
     description: Google service account name
     required: true
-  gcpManifestSigningKeyName:
-    description: Name of the manifest signing key in GCP KMS
-    required: false
   version:
     description: Version string (e.g., 30.0.0-rc1)
     required: true
   channel:
     description: Update channel
     required: false
     default: 'stable'
+  step:
+    description: Step to run
+    required: true
+    type: choice
```

Comment:
Is `type` valid for action inputs?

Reply:
Yes, it's not valid for reusable workflows apparently but here it works.